src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -126,4 +127,26 @@ public ResponseEntity findByIsbn(@RequestParam("isbn") String isbn) {
             return new ResponseEntity(book, HttpStatus.OK);
         }
     }
+
+    /**
+     * This method get a book by filter the Genre, Year and Publisher
+     *
+     * @param genre
+     * @param year
+     * @param publisher
+     * @return
+     */
+    @GetMapping()
```

Comment:
This route may need to be changed in a future, to be more specific about the required arguments